packages/babel-preset-react-app/create.js

```diff
@@ -54,6 +54,10 @@ module.exports = function (api, opts, env) {
     );
   }
 
+  const hasJsxRuntime = Boolean(
```

Comment:
nit & perhaps out of scope of this PR, but I noticed that we use a mixture of `var` and `const` here - is this intentional?

Reply:
It's legacy code, but I think most of us would use `const` today for newer code.

Reply:
Here's a PR for it: https://github.com/facebook/create-react-app/pull/9648

Reply:
Not intentional. I meant to change this to keep it consistent.